packages/web-components/src/contacts.ts

```diff
@@ -92,7 +93,7 @@ export const contactsMap = Object.freeze({
  * Map of phone numbers. CONTACTS.GI_BILL, for example, will return the phone
  * number defined in contactsMap.
  */
-export const CONTACTS = Object.freeze(
+export const CONTACTS: { [x: string]: string } = Object.freeze(
```

Comment:
Was this giving some Typescript errors before that this addition is resolving?

Reply:
Yes, previously it resolved as type `Readonly{ {} }` which meant that it didn't see any of the Object Keys for the ContactsMap.